### ISSUE TITLE
BndTools 3.0.0REL ignores subbundles?

### ISSUE THREAD

**Issue by Sandared** [bndtools/bndtools#1209]
Hi,

I encountered a strange behaviour when using BndTools 3.0.0.REL.
I created a new Project (test) and enabled subbundles. Then I created two additional Bundle Descriptors (testone, testtwo). Each one has a package added to its private section. Then I try to add both bundles in my launch.bndrun, but only the first one (test.testone) is offered by the add-dialogue. No matter how many other Bundle Descriptors I add, only the first one seems to be recognized.
If I add the test.testtwo bundle in the source-tab by hand it works, but still the "Available Bundles" section as well a the "add-dialogue" in the run tab only show the test.testone bundle.

Is this a known issue? I couldn't find anything like this in the open issues.

Kind regards,
Thomas

**Comment by pkriens** [comment]
Could you try restarting Eclipse?

**Comment by Sandared** [comment]
Hi Peter,

I restarted Eclipse, but this doesn't help. The jars are created correctly
in the "generated" folder of the project, but can not be found in the
"Available Bundles" section of the run tab in my launch.bndrun file.
Also in the repositories view only one of my subbundles can be found.

Kind regards,
Thomas

2015-11-13 19:28 GMT+01:00 Peter Kriens <notifications@github.com>:

**Comment by pkriens** [comment]
Can you share your workspace on github or dropbox?

**Comment by Sandared** [comment]
Hi Peter,

I've created a demoworkspace, which you can download here:
https://drive.google.com/file/d/0Bw1daHW2hcyZZEh4cFVPMlNOSnc/view?usp=sharing

Kind regards,
Thomas

2015-11-19 23:34 GMT+01:00 Peter Kriens <notifications@github.com>:

**Comment by pkriens** [comment]
Won’t have time this week, will look at this next week.

Kind regards,

	Peter Kriens

**Comment by Sandared** [comment]
Thanks for your effort :)


2015-11-23 9:21 GMT+01:00 Peter Kriens <notifications@github.com>:

> Won’t have time this week, will look at this next week.
>
> Kind regards,
>
> Peter Kriens
>

**Comment by pkriens** [comment]
I just verified the workspace (thanks for providing) and it does work for me on MacOS. We tested #1226 on Windows and there we also see a similar problem worked. 

I suspect the problem is that we're not tracking changes in the workspace repository (nor other repositories). We need a fix to automatically update the list of bundles & workspace repository view.

I am closing this as a duplicate of #1206 because I strongly suspect it is the same problem
